model/src/main/kotlin/RuleViolation.kt

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
Hint: The commit message still has ".......".

Reply:
Fixed.